pages/linux/trash.md

```diff
@@ -0,0 +1,23 @@
+# trash
+
+> A CLI trashcan/recycling bin.
```

Comment:
Perhaps - "A CLI for managing trash can / recycling bin" sounds better ?

Reply:
fixed